src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image-block {
+    display: flex;
+    margin: 32px 19px 40px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text-block {
+    margin: 0 $card-text-margin 4px;
+  }
+
+  &__text {
+    font-weight: $medium-font;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color-dark;
+  }
+
+  &__code-block {
+    margin: 0 $card-text-margin 16px;
+  }
+
+  &__code {
+    font-weight: $regular-font;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color-light;
+  }
+
+  &__reviews-block {
+    display: flex;
+    justify-content: space-between ;
+    align-items: center;
+    margin: 0 $card-text-margin 24px;
+  }
+
+  &__reviews {
+    font-weight: $regular-font;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color-dark;
```

Comment:
u can move `html` selector for this property, and use `inherit` where needed

Reply:
We have very different font sizes and font styles for the elements on this page. If I don't change, will it be an error?

Reply:
Do we have one which me use more often, then it make sense to move to `html`, if not u can leave it